sdk/ml/azure-ai-ml/azure/ai/ml/_exception_helper.py

```diff
@@ -143,47 +143,57 @@ def format_details_section(
     return error_types, details
 
 
-def format_errors_and_resolutions_sections(
-    entity_type: str, error_types: Dict[str, bool], cli: bool
-) -> Tuple[str, str]:
+def format_errors_and_resolutions_sections(error_types: Dict[str, bool], cli: bool) -> Tuple[str, str]:
     """Builds strings for details of the error message template's Errors and Resolutions sections."""
 
     resolutions = ""
     errors = ""
     count = 1
 
     if error_types[ValidationErrorType.INVALID_VALUE]:
-        errors += f"\n{count}) One or more fields are invalid"
+        errors += (
+            f"\n{count}) The yaml file you provided has one or more values" f" (for required fields) which are invalid:"
+        )
         resolutions += (
-            f"\n{count}) Double-check that all specified parameters are of the correct types and formats "
-            f"prescribed by the {entity_type} schema."
+            f"\n{count}) If the type is invalid, change it to the correct type. "
+            f"If value is invalid, choose a value from the expected range."
         )
         count += 1
     if error_types[ValidationErrorType.UNKNOWN_FIELD]:
-        errors += f"\n{count}) A least one unrecognized parameter is specified"
-        resolutions += f"\n{count}) Remove any parameters not prescribed by the {entity_type} schema."
+        errors += f"\n{count}) The yaml file you provided has one or more unrecognized parameters. "
```

Comment:
One nice-to-have suggestion would be to include the yaml file name in the error message. But that could make the formatting nasty if the user is using a very long name, so you would have to add some kind of truncation.